src/common/dom/dom.ts

```diff
@@ -0,0 +1,21 @@
+import {Renderer2, ElementRef} from '@angular/core';
+import {AppError} from 'ng2-qgrid/core/infrastructure';
+
+export class Dom {
```

Comment:
Bad name too common

Reply:
In my opinion this is redunded abstraction, if you don't wont to avoid copy paste by this, please make refactor it to composition(now is deriviation). Also be free to inject this code explicitly where it need.